lib/generators/migration/templates/migration_spec.rb.tt

```diff
@@ -0,0 +1,29 @@
+require_migration
+
+describe MoveMiqSetMemberships do
+  # let(:my_model_stub) { migration_stub(:MyModel) }
+
+  migration_context :up do
+    # Create data
```

Comment:
```suggestion
    # Create data
    #
    # Example:
    #
    #   my_model_stub.create!(attributes)
```

Reply:
👍

I think I stopped filling this out as it took me forever to figure out how to actually override it properly.